plugins/revealer/__init__.py

```diff
@@ -0,0 +1,17 @@
+from electrum.i18n import _
+
+fullname = _('Revealer')
+description = ''.join([
+    _("<br/><b>Do you have something to hide ?</b>"), '<br/>', '<br/>',
+    _("Revealer is a seed phrase back-up solution. It allows you to create a cold, analog, multi-factor backup of your wallet seeds, or of any arbitrary secret."), '<br/>', '<br/>',
+    _("Using a revealer is a better than writing your seed phrases on paper: a revealer is invulnerable to physical access and allows creation of trustless redundancy."), '<br/>', '<br/>',
```

Comment:
typo `is a better than`

Reply:
Thanks! 
Sorry, I'll fix the calibration sheet instructions, and qrcode dependency on PIL. 
On the 'assault' image, it looks like the included font (SansSource) did not load, any clues why?
About the code on the card: I might have to add another font. For now it's MONOSPACE_FONT defined by electrum, as it changes with the platform this issue. I hope it's ok as the font is around 200kb.